tests/backup/backup_resiliency_test.go

```diff
@@ -599,3 +599,156 @@ var _ = Describe("{RestartBackupPodDuringBackupSharing}", func() {
+			dash.VerifyFatal(srcClusterStatus, api.ClusterInfo_StatusInfo_Online, fmt.Sprintf("Verifying source cluster %s status", SourceClusterName))
+			destClusterStatus, _ = Inst().Backup.RegisterBackupCluster(orgID, destinationClusterName, "")
+			dash.VerifyFatal(destClusterStatus, api.ClusterInfo_StatusInfo_Online, fmt.Sprintf("Verifying destination cluster %s status", destinationClusterName))
+		})
+		Step("Taking backup of applications", func() {
+			log.InfoD("Taking backup of applications")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			dash.VerifyFatal(err, nil, "Getting admin context")
+			for _, namespace := range appNamespaces {
+				for i := 0; i < numberOfBackups; i++ {
+					backupName := fmt.Sprintf("%s-%s-%d", BackupNamePrefix, namespace, i)
+					backupNames = append(backupNames, backupName)
+					wg.Add(1)
+					go func(backupName string, namespace string) {
+						defer GinkgoRecover()
+						defer wg.Done()
+						defer func() { <-sem }()
+						_, err = CreateBackupWithoutCheck(backupName, SourceClusterName, bkpLocationName, backupLocationUID,
+							[]string{namespace}, labelSelectors, orgID, srcClusterUid, "", "", "", "", ctx)
+						dash.VerifyFatal(err, nil, fmt.Sprintf("Taking backup %s of application- %s", backupName, namespace))
+					}(backupName, namespace)
+				}
+			}
+			wg.Wait()
+			log.Infof("The list of backups taken are: %v", backupNames)
+			log.InfoD("Sleeping for 20 seconds so that the request reaches stork and the backup creation process is started")
+			time.Sleep(20 * time.Second)
+		})
+		Step("Cancelling the ongoing backups", func() {
+			log.InfoD("Cancelling the ongoing backups")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			dash.VerifyFatal(err, nil, "Getting admin context")
+			for _, backupName := range backupNames {
+				wg.Add(1)
+				go func(backupName string) {
+					defer GinkgoRecover()
+					defer wg.Done()
+					backupUID, err := Inst().Backup.GetBackupUID(ctx, backupName, orgID)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Getting UID for backup %v", backupName))
+					_, err = DeleteBackup(backupName, backupUID, orgID, ctx)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Deleting backup %s while backup is in progress", backupName))
+				}(backupName)
+			}
+			wg.Wait()
+			log.InfoD("Sleeping for 30 seconds for the backup cancellation to take place")
+			time.Sleep(30 * time.Second)
+		})
+		Step("Verifying if all the backup creation is cancelled", func() {
+			log.InfoD("Verifying if all the backup creation is cancelled")
+			adminBackups, err := GetAllBackupsAdmin()
+			log.Infof("The list of backups after backup cancellation is %v", adminBackups)
+			dash.VerifyFatal(err, nil, "Getting list of backups by px-central-admin")
+			if len(adminBackups) != 0 {
+				for _, backupName := range backupNames {
+					result := IsPresent(adminBackups, backupName)
+					dash.VerifyFatal(result, false, fmt.Sprintf("Verifying if backup %s is not present anymore", backupName))
```

Comment:
I do not see the verify fatal strings in the Aetos dashboard - http://aetos.pwx.purestorage.com/resultSet/testSetID/118748/testCaseID/581984

Has this been changed?

Reply:
It has not been changed.
The code is such that after cancellation, I fetch the list of backups.
If all the backups have been cancelled and there were no previous backups on the setup(if len(adminBackups) != 0 {}).. It will not go inside this loop and print : log.Infof("All the backups created by this testcase is deleted after backup cancellation")

Reply:
https://jenkins.pwx.dev.purestorage.com/job/Users/job/Sumit/job/Custom-Pipelines/job/px-backup-on-demand-system-test-byoc/260/consoleFull
In the above link, I have purposely created a backup "test-debug"..in this case we can see the verify fatal strings